src/main/resources/assets/supercircuitmaker/lang/de_de.json

```
@@ -0,0 +1,51 @@
+{
+  "itemGroup.supercircuitmaker": "Super Circuit Maker",
+
+  "block.supercircuitmaker.circuit": "Schaltkreis",
+  "block.supercircuitmaker.inspector": "Inspektor",
+  "item.supercircuitmaker.screwdriver": "Schraubendreher",
+  "item.supercircuitmaker.palette": "Palette und Pinsel",
+  "item.supercircuitmaker.tiny_redstone": "Kleiner Haufen von Redstone Staub",
+  "item.supercircuitmaker.tiny_rgb_redstone": "Kleiner Haufen von buntem Redstone Staub",

```

```suggestion
  "item.supercircuitmaker.tiny_redstone": "Kleiner Haufen von Redstone-Staub",
  "item.supercircuitmaker.tiny_rgb_redstone": "Kleiner Haufen von buntem Redstone-Staub",
```

Maybe also *Häufchen* to make it shorter? Or even
```suggestion
  "item.supercircuitmaker.tiny_redstone": "Häufchen Redstone-Staub",
  "item.supercircuitmaker.tiny_rgb_redstone": "Häufchen bunter Redstone-Staub",
```
which should also be correct, I believe?